### ISSUE TITLE
Log warning for missing translation

### ISSUE THREAD

**Issue by jbmoelker** [voorhoede/head-start#129]
If there is no translation available for a given translation key the [`t` function from `lib/i18n.ts`](https://github.com/voorhoede/head-start/blob/d8b6aba09ebe06c4c19b8f978efd2cfed7d3c894/src/lib/i18n.ts#L14) will return an empty string `''`. Some translations may be used for visually hidden content like `aria-labels`, where it will be extra difficult to detect missing translations. Therefore it would be a good enhancement to log a warning for missing translations.
